en/activity-standard/iati-activities/iati-activity/budget.rst

```diff
@@ -30,4 +30,7 @@ Changelog
 
 2.02
 ^^^^
+
 The ``status`` attribute was `added <http://support.iatistandard.org/entries/21150501-Budgets-and-tentativeness>`__.
+
+The ``budget`` definition was updated as a `bug fix <https://discuss.iatistandard.org/t/proposal-update-of-budget-element-description-text-in-activity-standard/1216>`__ on the 27/02/2018.
```

Comment:
I would be more tempted to include this above the heading for `2.02`.

We could have the date of the change as the section header?